src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/CaseProceedOfflineDefendantNotificationHandler.java

```diff
@@ -67,4 +71,30 @@ public boolean shouldRecordScenarioInCaseProgression(CaseData caseData, boolean
             && caseMovedInCaseManStatesCaseProgression.contains(caseData.getPreviousCCDState())
             && isLipvLipOrLRvLip;
     }
+
+    @Override
+    protected void beforeRecordScenario(CaseData caseData, String authToken) {
+        dashboardApiClient.deleteNotificationsForCaseIdentifierAndRole(
```

Comment:
This would remove all notifications from the case, whether or not they relate to G.A. Is this correct?